### PR TITLE
NS-445 Move metadata schema from Redshift to RDS

### PR DESCRIPTION
**Pull request by raydavis**
https://jira.ets.berkeley.edu/jira/browse/NS-445

### CODE REVIEW

File: scripts/migrate/20190410-NS-445/copy_rds_metadata_from_redshift.sql [hunk 1]
```diff
@@ -0,0 +1,125 @@
+-- This script needs the "redshift_db" variable set before being run.
```
Comment (raydavis) [inline scripts/migrate/20190410-NS-445/copy_rds_metadata_from_redshift.sql]: This seemed a bit more in line with our release conventions than writing a shell script, but let me know if you disagree.

File: nessie/jobs/create_metadata_schema.py [hunk 2]
```diff
@@ -44,11 +44,4 @@ def run(self):
             raise BackgroundJobError(f'RDS metadata schema creation failed.')
             return False
 
-        resolved_ddl_redshift = resolve_sql_template('create_metadata_schema_redshift.template.sql')
```
Comment (johncrossman) [inline nessie/jobs/create_metadata_schema.py]: @raydavis - Is `create_metadata_schema_redshift.template.sql` still used?

File: scripts/migrate/20190410-NS-445/copy_rds_metadata_from_redshift.sql (hunk 1 above)
Comment (johncrossman) [reply scripts/migrate/20190410-NS-445/copy_rds_metadata_from_redshift.sql]: I agree.

File: nessie/jobs/create_metadata_schema.py (hunk 2 above)
Comment (raydavis) [reply nessie/jobs/create_metadata_schema.py]: Not by code, @johncrossman , so I should delete it, even if the Redshift config variables stay around for a release. I'll amend.

File: tests/conftest.py [hunk 3]
```diff
@@ -107,9 +108,12 @@ def metadata_db(app):
         created_at TIMESTAMP NOT NULL,
         deleted_at TIMESTAMP
     )""")
-    rds_schema = app.config['RDS_SCHEMA_METADATA']
-    rds.execute(f'DROP SCHEMA IF EXISTS {rds_schema} CASCADE')
-    rds.execute(f'CREATE SCHEMA IF NOT EXISTS {rds_schema}')
+    rds.execute(f"""CREATE TABLE IF NOT EXISTS {rds_schema}.merged_feed_status
+    (
+        sid VARCHAR NOT NULL PRIMARY KEY ,
```
Comment (johncrossman) [inline tests/conftest.py]: @raydavis Minor: remove space before comma

**Comment by johncrossman** [comment]
@raydavis Looks good. Can we remove `REDSHIFT_SCHEMA_METADATA` from `config/*.py` files?

**Comment by raydavis** [comment]
@johncrossman , I had planned to keep the REDSHIFT_SCHEMA_METADATA configs in place so as to support copying old Redshift data to the new RDS tables. However, since we're going to try doing the copy with plain SQL (plus one variable), I think we can remove them after all.

I don't yet know what's up with Travis, since local runs work for me. But I'll keep poking at it for a bit.

**Comment by raydavis** [comment]
@johncrossman Looks like that last amendment pleased Travis.

**Comment by raydavis** [comment]
... but we'll hold off on a merge until tomorrow morning, to avoid conflicts with nightly jobs.

**Pull request merged**